remote-api/src/test/java/org/alfresco/rest/api/TagsUnitTests.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2023 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api;
+
+import org.alfresco.rest.api.impl.TagsImplTest;
+import org.alfresco.rest.api.tags.TagsEntityResourceTest;
+import org.alfresco.service.Experimental;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@Experimental
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    TagsImplTest.class,
+    TagsEntityResourceTest.class
+})
+public class TagsUnitTests
```

Comment:
I don't immediately see the value in this test suite. Do you think it will be useful to run just this combination of tests in the future?

Reply:
Those are newly added unit tests (which can be extended in future) so I though that they could be grouped somehow. I've also added `TaggingServiceImplUnitTest` but it's in a different package: `org.alfresco.repo.tagging`, so I didn't placed it in this tests suite (package `org.alfresco.rest.api`).